test/karma.conf.js

```diff
@@ -133,43 +133,6 @@ module.exports = function setKarmaConfig(config) {
               envName: 'stable',
             },
           },
-          // transpile 3rd party packages with dependencies in this repository
```

Comment:
The tests are passing without it
I think it is not needed since we don't have pickers tests on the core anymore

Reply:
We need it again for the tree view :laughing: